content/docs/command-reference/add.md

```diff
@@ -336,91 +339,78 @@ $ tree .dvc/cache
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
 
-## Example: Transfer to the cache
+## Example: Caching large data externally {#straight-to-cache}
 
-When you have a large dataset in an external location, you may want to add it to
-the <abbr>project</abbr> without having to copy it into the workspace. Maybe
-your local disk doesn't have enough space, but you have setup an
-[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-that could handle it.
+Sometimes you may want to add a large dataset currently found in an external
+location, so it becomes local to the project. But what if there's not enough
+disk space to download the data?
 
-The `--out` option lets you add external paths in a way that they are
+The `--out` option lets you add external data in a way that it's
 <abbr>cached</abbr> first, and then
 [linked](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
-to a given path inside the <abbr>workspace<abbr>. Let's initialize an example
-DVC project to try this:
-
-```dvc
-$ mkdir example # workspace
-$ cd example
-$ git init
-$ dvc init
-```
+to a given path inside the <abbr>workspace</abbr>. Combined with an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+and the right
+[kind of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache),
+this let's you avoid using the local file system completely.
 
-Now we can add a `data.xml` file via HTTP for example, putting it a local path
-in our project:
+For example, we can add a `data.xml` file via HTTP, outputting it to a local
+path in our project:
 
 ```dvc
-$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml
+$ dvc add https://data.dvc.org/get-started/data.xml -o raw/data.xml
 
 $ ls
 data.xml data.xml.dvc
 ```
 
-The resulting `.dvc` file will save the provided local `path` as if the data was
-already in the workspace, while the `md5` hash points to the copy of the data
-that has now been transferred to the <abbr>cache</abbr>. Let's check the
-contents of `data.xml.dvc` in this case:
+The local `data.xml` should be a symlink or hard link to the externally
+<abbr>cached</abbr> data copy. The resulting `.dvc` file will save the local
+`path` as if the data was already there before this command. Let's check the
+contents of `data.xml.dvc`:
 
 ```yaml
 outs:
   - md5: a304afb96060aad90176268345e10355
     nfiles: 1
-    path: data.xml
+    path: raw/data.xml
 ```
 
 > For a similar operation that actually keeps a connection to the data source,
 > please see `dvc import-url`.
 
-## Example: Transfer to remote storage
+## Example: Storing large data remotely {#straight-to-remote}
```

Comment:
name is also still confusing, doesn't reflect an actual use case (we store data remotely anyway). This is about adding data ... should be something related to that

Reply:
> we store data remotely anyway

True. Let me think ⌛ 

> This is about adding data

It's the `add` reference so no nee to use the word "add" in every example title, I think.

Reply:
> we store data remotely anyway

Actually, while true, the context here is not the `push` or `pull` commands, so storing data remotely is not necessarily assumed. Remote storage isn't even mentioned in the main Description.

But anyway, how about "Bootstrap external data tracking with remote storage"? (applied for now) It's a bit long though...

Reply:
-> https://github.com/iterative/dvc.org/pull/2302#issuecomment-801442953
Resolving